components/external-link/style.scss

```diff
@@ -0,0 +1,7 @@
+.new-window-icon {
+	.dashicon {
+		width: 1.4em;
+		height: 1.4em;
+		vertical-align: top;
+	}
+}
```

Comment:
We don't have checks for it, but: https://github.com/WordPress/gutenberg/blob/v0.3.0/.editorconfig#L9. Is there any way to add checks @nylen or @ntwb?